app/bundles/EmailBundle/Views/Email/form.html.twig

```diff
@@ -0,0 +1,279 @@
+                                        {% endfor %}
+                                        {% set tabHtml = tabHtml ~ '</ul></div>' %}
+                                        {% set tabContentHtml = tabContentHtml ~ '</div>' %}
+
+                                        {{ tabHtml }}
+                                        {{ tabContentHtml }}
+                                    </div>
+                                </div>
+                            </div>
+                        </div>
+                        {{ customContent('email.tabs.content', mauticTemplateVars) }}
+                    </div>
+                </div>
+            </div>
+        </div>
+        <div class="col-md-3 bg-white height-auto bdr-l">
+            <div class="pr-lg pl-lg pt-md pb-md">
+                {{ form_row(form.subject) }}
+                {{ form_row(form.name) }}
+                {% if isVariant %}
+                    {{ form_row(form.variantSettings) }}
+                    {{ form_row(form.isPublished) }}
+                    {{ form_row(form.publishUp) }}
+                    {{ form_row(form.publishDown) }}
+                {% else %}
+                    <div id="leadList"{{ (('template' == emailType) ? ' class="hide"' : '') }}>
+                        {{ form_row(form.lists) }}
+                    </div>
+                    {{ form_row(form.category) }}
+                    {{ form_row(form.language) }}
+
+                    <div id="segmentTranslationParent"{{ (('template' == emailType) ? ' class="hide"' : '') }}>
+                        {{ form_row(form.segmentTranslationParent) }}
+                    </div>
+                    <div id="templateTranslationParent"{{ (('list' == emailType) ? ' class="hide"' : '') }}>
+                        {{ form_row(form.templateTranslationParent) }}
+                    </div>
+                {% endif %}
+
+                {% if not isVariant %}
+                    {{ form_row(form.isPublished) }}
+                    {{ form_row(form.publishUp) }}
+                    {{ form_row(form.publishDown) }}
+                {% endif %}
+
+                {{ form_row(form.unsubscribeForm) }}
+                {% if permissions['page:preference_center:viewown'] is not empty and permissions['page:preference_center:viewother'] is empty %}
+                    {{ form_row(form.preferenceCenter) }}
+                {% endif %}
+                <hr />
+                <h5>{{ mautic.email.utm_tags|trans }}</h5>
+                <br />
+                {% for i, utmTag in form.utmTags %}
+                    {{ form_row(utmTag) }}
+                {% endfor %}
+            </div>
+            <div class="hide">
+                {{ form_rest(form) }}
+            </div>
+        </div>
+    </div>
+    {{ form_end(form) }}
+    <div id="dynamicContentPrototype" data-prototype="{{ form_widget(dynamicContentPrototype)|escape }}"></div>
+    {% if instanceof(filterBlockPrototype, FormView)%}
```

Comment:
The `instanceof` check is not a Twig function but a Twig test. See example usage: https://github.com/mautic/mautic/blob/5.x/app/bundles/CoreBundle/Views/Standard/list.html.twig#L84

Reply:
Thanks! There are still some failed tests. Please take a look